es6.js

```diff
@@ -26,9 +26,9 @@ module.exports = {
     "no-useless-rename": 0,
     "no-var": 2,
     "babel/object-shorthand": 0,
-    "prefer-arrow-callback": [1, { "allowNamedFunctions": true }],
+    "prefer-arrow-callback": 0,
     "prefer-const": 1,
-    "prefer-reflect": 1,
+    "prefer-reflect": 0,
```

Comment:
Personally, I don't want to use `Reflect` instead of `Function.prototype.call` :)